cluster_config.go

```diff
@@ -194,6 +199,9 @@ func (cfg *Config) Validate() error {
 
 // this just sets non-generated defaults
 func (cfg *Config) setDefaults() {
+	hostname, _ := os.Hostname()
```

Comment:
For politeness and explicitness, read the error, and if it's not nil, set to empty string.